patches/server/0983-Array-backed-synched-entity-data.patch

```diff
@@ -0,0 +1,55 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Warrior <50800980+Warriorrrr@users.noreply.github.com>
```

Comment:
Author should be the original code author with a link to the license in the patch description